### ISSUE TITLE
Slow StopsController include query

### ISSUE THREAD

**Issue by irees** [transitland/transitland-datastore#996]
Slow performance in https://github.com/transitland/transitland-datastore/blob/master/app/controllers/api/v1/stops_controller.rb#L60-L69 under certain conditions, such as when using served_by operator.

Example:
https://transit.land/api/v1/stops.geojson?served_by=o-c2b8-britishcolumbiarapidtransitcompany

Can time out. Checked on the console, the include query takes about 60s to run.

**Comment by irees** [comment]
```Stop.served_by(['r-9q8yw-44']).includes{[
  operators_serving_stop,
  operators_serving_stop.operator,
  routes_serving_stop,
  routes_serving_stop.route,
  routes_serving_stop.route.operator,
  imported_from_feeds,
  imported_from_feed_versions
]}.to_a.size```

Takes about 15s -- however, it depends on the combination of `includes`. Without `imported_from_feeds` and `imported_from_feed_versions`, returns almost immediately. However, it isn't the `imported_from_feeds...` includes themselves that make it slow, as with just these two `includes` returns immediately...

Possible idea: move `imported_from` queries to a separate controller.

**Comment by drewda** [comment]
We'll only include `imported_from_feeds` and `imported_from_feed_versions` when `?embed_imported_from_feed_info=true` or `?imported_with_gtfs_id=...`.

**Comment by irees** [comment]
Resolved by #1096 

`?include=imported_from_feeds`
